api/hub/service.go

```diff
@@ -204,50 +204,67 @@ func getSessionSecret(secret string) string {
 	return util.MakeToken(44)
 }
 
-func (s *Service) Signout(ctx context.Context, _ *pb.SignoutRequest) (*pb.SignoutReply, error) {
+func (s *Service) Signout(ctx context.Context, _ *pb.SignoutRequest) (*pb.SignoutResponse, error) {
 	log.Debugf("received signout request")
 
 	session, _ := mdb.SessionFromContext(ctx)
 	if err := s.Collections.Sessions.Delete(ctx, session.ID); err != nil {
 		return nil, err
 	}
-	return &pb.SignoutReply{}, nil
+	return &pb.SignoutResponse{}, nil
 }
 
-func (s *Service) GetSessionInfo(ctx context.Context, _ *pb.GetSessionInfoRequest) (*pb.GetSessionInfoReply, error) {
+func (s *Service) GetSessionInfo(ctx context.Context, _ *pb.GetSessionInfoRequest) (*pb.GetSessionInfoResponse, error) {
 	log.Debugf("received get session info request")
 
 	dev, _ := mdb.DevFromContext(ctx)
 	key, err := crypto.MarshalPublicKey(dev.Key)
 	if err != nil {
 		return nil, err
 	}
-	return &pb.GetSessionInfoReply{
+	return &pb.GetSessionInfoResponse{
 		Key:      key,
 		Username: dev.Username,
 		Email:    dev.Email,
 	}, nil
 }
 
-func (s *Service) CreateKey(ctx context.Context, req *pb.CreateKeyRequest) (*pb.GetKeyReply, error) {
+func (s *Service) CreateKey(ctx context.Context, req *pb.CreateKeyRequest) (*pb.CreateKeyResponse, error) {
 	log.Debugf("received create key request")
 
 	owner := ownerFromContext(ctx)
-	key, err := s.Collections.APIKeys.Create(ctx, owner, mdb.APIKeyType(req.Type), req.Secure)
+
+	var keyType mdb.APIKeyType
+	switch req.Type {
```

Comment:
Treating all conversion to/from proto enums explicitly is a good practice. I added a "unspecified" value in the 0 position of each of them, so the number doesn't always align with our native Go representation, which isn't ever a good thing to assume anyway.

Reply:
Made similar changes elsewhere